### ISSUE TITLE
migration to puppet version 5

### ISSUE THREAD

**Issue by pmelange** [freifunk-berlin/puppet#58]
On newer distributions (ubuntu 18, debian 9) puppet 5 is used.  It would be good to migrate the puppet configurations to work with these newer distributions.

**Comment by pmelange** [comment]
It is not necessary to migrate to puppet5.  The install script needs to be updated
https://github.com/freifunk-berlin/puppet/blob/ae93143ae75a561e4f9fc9e4dbaeb0014077c575/scripts/install.sh#L3-L5

to use something like (adding the lsb_release of course): 
```
wget http://apt.puppetlabs.com/puppet-release-xenial.deb
dpkg -i /tmp/puppet-release-xenial.deb
```

**Comment by pmelange** [comment]
Fixed in #66
